README.md

```diff
@@ -55,33 +55,63 @@ AES is recommended for shadowsocks-go. [Intel AES Instruction Set](http://en.wik
 
 **rc4 and table encryption methods are deprecated because they are not secure.**
 
-### One Time Auth
-
-Append `-auth` to the encryption method to enable [One Time Auth (OTA)](https://shadowsocks.org/en/spec/one-time-auth.html).
-
-- For server: this will **force client use OTA**, non-OTA connection will be dropped. Otherwise, both OTA and non-OTA clients can connect
-- For client: the `-A` command line option can also enable OTA
-
-### UDP relay
-
-Use `-u` command line options when starting server to enable UDP relay.
-
-Currently only tested with Shadowsocks-Android, if you have encountered any problem, please report.
-
 ## Command line options
 
 Command line options can override settings from configuration files. Use `-h` option to see all available options.
+ss-local
+```
+Usage of ./shadowsocks-local:
+  -addr string
+        local socks5 proxy serve address (default "127.0.0.1")
+  -config string
+        specify config file
+  -level string
+        given the logger level for ss to logout info, can be set in debug info warn error panic (default "info")
+  -method string
+        encryption method, default: aes-256-cfb. end with -auth mean enable OTA (default "aes-256-cfb")
```

Comment:
This line need to be modified, the ota things.

Reply:
thanks for reply, this will be removed

Reply:
@dszhengyu  fixed in 8796767